sdk/eventgrid/azure-eventgrid/tests/recordings/test_eg_publisher_client.test_send_cloud_event_data_with_extensions.yaml

```diff
@@ -0,0 +1,40 @@
+interactions:
+- request:
+    body: '[{"id": "9883be2c-2252-4c42-bb32-31317b72237f", "source": "http://samplesource.dev",
+      "data": "cloudevent", "type": "Sample.Cloud.Event", "time": "2020-09-03T08:23:07.846089Z",
```

Comment:
As you can see the extension is not there, but with the additional_property kwarg you should have them now